src/amo/components/TooltipMenu/styles.scss

```diff
@@ -73,7 +73,7 @@
     button {
       overflow: hidden;
       text-overflow: ellipsis;
-      white-space: nowrap;
+      white-space: normal;
```

Comment:
I guess it's only a problem for small screen sizes anyway, because it won't ever be long enough to wrap for larger sizes.

Reply:
oh, mid-air with Bob's comments.  I didn't see that when I had a quick play

Reply:
> I tested it locally and it seems fine to apply it to all screen sizes, but I am still seeing truncation at some small screen sizes. See the attached screenshot. It would be nice if this could be fixed as well.

I can reproduce and this is because something in the header breaks the layout, which is unrelated to this issue.